app/controllers/v0/facilities/ccp_controller.rb

```diff
@@ -10,6 +10,7 @@ def show
     caresites = ppms.provider_caresites(params[:id])
     result.Latitude = caresites[0]['Latitude'] unless caresites.nil?
```

Comment:
now that you are stacking multiple of these with the same condition, consider moving away from the inline approach and to a block. I'd also switch to an `if caresites` or `if caresites.present?`